### PR TITLE
readme file has been added

### PR DESCRIPTION
**Pull request by ekizanlikli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Reddit Client
+![CI/CD](https://github.com/ekizanlikli/reddit-client/workflows/Node.js%20CI/badge.svg)
+[![codecov](https://codecov.io/gh/ekizanlikli/reddit-client/branch/main/graph/badge.svg)](https://codecov.io/gh/ekizanlikli/reddit-client)
+![License](https://img.shields.io/github/license/ekizanlikli/reddit-client)
+
+
+## To-dos
+[ ] Write unit tests for your components using Jest and Enzyme
```
Comment (ahmetaydin-aa) [inline README.md]: Checkboxes not working, please fix it.

**Review by ahmetaydin-aa — CHANGES_REQUESTED** [review]
Leaving some comments to change.

File: README.md (hunk 1 above)
Comment (ekizanlikli) [reply README.md]: The issue has been resolved

**Review by ahmetaydin-aa — APPROVED** [review]
LGTM👍🏻

**Pull request merged**